scripts/updateLicense.py

```diff
@@ -17,10 +17,15 @@
     ('// ' + l).strip() + '\n' for l in LICENSE_BLOB.split('\n')
 ]
 
+LICENSE_BLOB_LINES_SHELL_AND_MAKEFILE = [
```

Comment:
Change to a function that accepts comment prefix

Reply:
got it, i was thinking that too